examples/cactus-example-carbon-accounting-backend/src/utility-emissions-channel/typescript/src/index.ts

```diff
@@ -575,4 +575,5 @@ export class EmissionsChaincode {
   }
 }
 
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
@akaashsaxena2897 If you disable the linter please also attach an explanation detailing why it was not feasible/impossible to fix the underlying error instead of turning off the linter for the line. (if no such explanation exists then the recommended course of action is to fix the linter warning instead of turning off the linter)

Reply:
@akaashsaxena2897 Next time please tag me otherwise it doesn't show up on my GH feed

Reply:
@akaashsaxena2897 

> To remove the linter it can be written like this
> export const contracts: typeof EmissionsChaincode[] = [EmissionsChaincode];
> I'd personally prefer following the syntax given in the document to , and disable linter for the line.

I'd very much prefer that as well because that way we don't need to disable the linter at all. Is there any reason why not go with that solution? I see in your diff that you still have the linter disabled and the any keyword in usage.